GestureHandler.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
 import {
   Animated,
-  ScrollView,
-  Switch,
-  TextInput,
-  ToolbarAndroid,
-  DrawerLayoutAndroid,
   StyleSheet,
-  FlatList,
   Platform,
   processColor,
+  default as ReactNative,
```

Comment:
This will include everything in React Native which means things like react-art which aren't used by anyone will get bundled.